docker/src/main/java/org/jclouds/docker/features/ImageApi.java

```diff
@@ -102,4 +104,23 @@
    @Path("/images/{name}")
    InputStream deleteImage(@PathParam("name") String name, DeleteImageOptions options);
 
+   /**
+    * Tag the image name into a repository.
+    *
+    * @param name
+    *           the name of the image to be tagged
+    * @param repoName
+    *           the repository to tag in
+    * @param tag
+    *           the new tag name
+    * @param force
+    *           force create if tag already exists
+    * @return true if tag is created, false when image is not found.
+    */
+   @Named("image:tag")
+   @POST
+   @Path("/images/{name}/tag")
+   @Fallback(FalseOnNotFoundOr404.class)
```

Comment:
In general we try to avoid using 404 fallbacks in POST/PUT methods, as often they don't provide the right feedback to the user. Operations done on objects that are supposed to exist will already [throw a ResourceNotFoundException](https://github.com/jclouds/jclouds-labs/blob/master/docker/src/main/java/org/jclouds/docker/handlers/DockerErrorHandler.java#L71-L75). I'd change this method to return `void` and remove the fallback.